site/pt/blog/new-in-devtools-111/index.md

```diff
@@ -27,31 +26,41 @@ draft: true
+O painel **Estilos** agora oferece suporte a 12 novos espaços de cores e 7 novas gamas, conforme descrito na especificação [CSS Color Level 4](https://www.w3.org/TR/css-color-4/). Consulte o [Guia de cores CSS de alta definição](/articles/high-definition-css-color-guide/#debugging-color-with-chrome-devtools) para obter uma compreensão abrangente das opções de cores disponíveis na Web.
 
 <!-- Here are examples of CSS color definitions with `color()`, `lch()`, `oklab()` and `color-mix()`. -->
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/dA8VCKaSZhNb9gzlAUT9.png", alt="Examples of CSS color definitions.", width="800", height="509" %}
+Aqui estão exemplos de definições de cores CSS com `color()`, `lch()`, `oklab()` e `color-mix()`.
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/dA8VCKaSZhNb9gzlAUT9.png", alt="Exemplos de definição de cores com CSS", width="800", height="509" %}
 
 <!-- When using the `color-mix()` function, you can view the final color output in the **Computed** pane. -->
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3VkOGbbb5qLVvo1A1qSa.png", alt="color-mix result in the Computed pane.", width="800", height="487" %}
+Ao usar a função `color-mix()`, você pode visualizar a saída de cor final no painel **Computed**.
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/3VkOGbbb5qLVvo1A1qSa.png", alt="Resultado do color-mix no painel Computed.", width="800", height="487" %}
 
 <!-- The color picker supports all the new color spaces with more features. For example, click on the color swatch of `color(display-p3 1 0 1)`. A gamut boundary line has also been added, distinguishing between the `sRGB` and `display-p3` gamuts for a clearer understanding of your selected color's gamut. -->
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bL6uw8VV4cGuDd9hmAjX.png", alt="A gamut boundary line.", width="800", height="657" %}
+O seletor de cores suporta todos os novos espaços de cores com mais recursos. Por exemplo, clique na amostra de cor de `color(display-p3 1 0 1)`. Uma linha de limite de gama também foi adicionada, distinguindo entre as gamas `sRGB` e `display-p3` para uma compreensão mais clara da gama de cores selecionada.
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/bL6uw8VV4cGuDd9hmAjX.png", alt="A linha de distinção do gama.", width="800", height="657" %}
 
 <!-- DevTools supports converting colors between color formats. Use the **Change Color Format** icon to access the conversion popup, or simply use the `Shift` + click on a color swatch in the **Styles** pane. -->
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/uoz3yaPPdVs6T2ASnQ62.png", alt="Converting colors between color formats.", width="800", height="460" %}
+O DevTools oferece suporte à conversão de cores entre formatos de cores. Use o ícone **Alterar formato de cor** para acessar o pop-up de conversão ou simplesmente use `Shift` + clique em uma amostra de cor no painel **Estilos**.
```

Comment:
nit: "O DevTools oferece suporte à conversão de cores entre formatos diferentes.' The current translation aligns with the original text, but it looks slightly better in pt_BR if we avoid the repetition.

Reply:
```suggestion
O DevTools oferece suporte à conversão de cores entre formatos diferentes. Use o ícone **Alterar formato de cor** para acessar o pop-up de conversão ou simplesmente use `Shift` + clique em uma amostra de cor no painel **Estilos**.
```